modules/internal/ChapelDomain.chpl

```diff
@@ -1852,6 +1852,9 @@ module ChapelDomain {
       return _value.dsiBulkAdd(inds, dataSorted, isUnique, preserveInds, addOn);
     }
 
+    deprecated "makeIndexBuffer has been renamed to createIndexBuffer"
```

Comment:
I'd add a new deprecation test to /test/deprecated/ for `makeIndexBuffer`. (It's not super important, but those tests are nice to have so that we don't lose track of deprecated symbols.)

Reply:
Good catch, I meant to do that and forgot